src/test/java/ua/goit/gojava32/kickstarter/test/EqualsTest.java

```diff
@@ -0,0 +1,35 @@
+package ua.goit.gojava32.kickstarter.test;
+
+import org.junit.Test;
+import ua.goit.gojava32.kickstarter.model.User;
+
+import static org.junit.Assert.assertEquals;
+
+public class EqualsTest {
```

Comment:
usually equals / hashcode are generated on demand by your IDE so testing them is not needed